### PR TITLE
Urgent fix to show user location on Task Manager

### PR DESCRIPTION
**Pull request by connorpgpmcelroy**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3828

### Change description ###
Request to reduce the backend load by ensuring that any user will be shown the Task Manager page with the location filter defaulted to their base location (where possible).

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by udaydenduluri33 — DISMISSED** [review]
LGTMe

**Review by leepavlouhmcts — APPROVED** [review]
approved

**Review by udaydenduluri33 — APPROVED** [review]
LGTM after the merge from master

File: src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts
Comment (ronaldmansveld) [inline src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts:94]: Should never use any. Either make changes in the rest of the code so it's always a string, or, if that can't be done, make this a compound type.

File: src/work-allocation/containers/task-manager-list/task-manager-list.component.ts
Comment (ronaldmansveld) [inline src/work-allocation/containers/task-manager-list/task-manager-list.component.ts:65]: Why the prefix 'p' for this public property?

**Review by ronaldmansveld — CHANGES_REQUESTED** [review]

**Review by leepavlouhmcts — APPROVED** [review]

File: src/work-allocation/containers/task-manager-list/task-manager-list.component.ts
Comment (udaydenduluri33) [reply src/work-allocation/containers/task-manager-list/task-manager-list.component.ts]: fixed the issues

File: src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts
Comment (udaydenduluri33) [reply src/work-allocation/components/task-manager-filter/task-manager-filter.component.ts]: Agreed, changed it to string

File: src/work-allocation/containers/task-manager-list/task-manager-list.component.ts
Comment (udaydenduluri33) [reply src/work-allocation/containers/task-manager-list/task-manager-list.component.ts]: Done

**Review by leepavlouhmcts — APPROVED** [review]

**Review by ronaldmansveld — APPROVED** [review]

**Pull request merged**
